core/src/components/checkbox-group/checkbox-group.stories.js

```diff
@@ -0,0 +1,199 @@
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 1</div>
+        </div>
+      </label>
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 2</div>
+        </div>
+      </label>
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 3</div>
+        </div>
+      </label>
+    </div>
+  </fieldset>
+`;
+
+export const Guidance = () => `
+  <fieldset class="fudis-checkbox-group fudis-fieldset fudis-input-size__lg">
+    <legend class="fudis-fieldset__legend">
+      <div class="fudis-fieldset__legend__main fudis-fieldset__legend__sm">
+        <div class="fudis-fieldset__legend__main__text-content">
+          <span class="fudis-fieldset__legend__main__text">
+            Legend
+          </span>
+        </div>
+      </div>
+    </legend>
+    <div id="fudis-fieldset-id" class="fudis-fieldset-content">
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 1</div>
+        </div>
+      </label>
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 2</div>
+        </div>
+      </label>
+      <label class="fudis-checkbox">
+        <input type="checkbox" class="fudis-checkbox__input"></input>
+        <div class="fudis-checkbox__content">
+          <div class="fudis-checkbox__content-wrapper">
+            <span class="fudis-checkbox__content__box"></span>
+          </div>
+          <div class="fudis-checkbox__content__label">Option 3</div>
+        </div>
+      </label>
+      <div id="fudis-guidance-id" class="fudis-guidance">
```

Comment:
For all the examples that have guidance present, we could link guidance id with aria-describedby attribute to be used in a first input element.

Reply:
Added `aria-describedby` to CheckboxGroup and RadioButtonGroup's _Guidance_ and _Invalid_ stories.